src/System.ComponentModel.TypeConverter/tests/ToolboxItemAttributeTests.cs

```diff
@@ -89,7 +89,7 @@ public void Default_Get_ReturnsExpected()
             ToolboxItemAttribute attribute = ToolboxItemAttribute.Default;
             Assert.Same(attribute, ToolboxItemAttribute.Default);
             Assert.Equal("System.Drawing.Design.ToolboxItem, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", attribute.ToolboxItemTypeName);
-            if (!PlatformDetection.IsUap)
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
This has to be: `if (PlatformDetection.IsFullFramework)` -- that will fix the failures. The type exists in full framework so it will be not null. Right now the test is doing the opposite, expecting the type to not exist in full framework and to exist in all other frameworks.

Reply:
Thanks.